Version 1.7.2 (#3797)
Kotlin/kotlinx.coroutines: CHANGES.md

# Change log for kotlinx.coroutines

## Version 1.7.2

### Bug fixes and improvements

* Coroutines debugger no longer keeps track of coroutines with empty coroutine context (#3782).

> * `CopyableThreadContextElement` is now properly copies an element when crossing coroutines boundary in `flowOn` operator (#3787). Thanks @wanyingd1996!

Reviewer: ```suggestion
* `CopyableThreadContextElement` now properly copies an element when crossing the coroutine boundary in `flowOn` (#3787). Thanks @wanyingd1996!
```
